response_operations_ui/views/collection_exercise.py

```diff
@@ -878,12 +978,16 @@ def get_seft_collection_instrument(short_name, period):
         info_panel=info_panel,
         show_msg=show_msg,
         period=period,
+        locked=locked,
     )
 
 
 @collection_exercise_bp.route("/<short_name>/<period>/load-collection-instruments", methods=["POST"])
 @login_required
 def post_seft_collection_instrument(short_name, period):
+    if not uaa_controller.user_has_permission("surveys.edit"):
+        logger.error("User has insufficient permissions to access this page", user_id=session["user_id"])
```

Comment:
Looking at this again, I think you can just remove the logger. The route with the 403 tells us everything